robottelo/utils/sso.py

```diff
@@ -0,0 +1,218 @@
+import json
+import random
+from contextlib import contextmanager
+
+from broker.hosts import Host
+from fauxfactory import gen_string
+from pexpect import pxssh
+
+from robottelo.config import settings
+from robottelo.constants import KEY_CLOAK_CLI
+from robottelo.constants import RHSSO_NEW_GROUP
+from robottelo.constants import RHSSO_NEW_USER
+from robottelo.constants import RHSSO_RESET_PASSWORD
+from robottelo.constants import RHSSO_USER_UPDATE
+from robottelo.datafactory import valid_emails_list
+
+
+class SSOHost(Host):
+    def __init__(self, **kwargs):
+        self.hostname = kwargs.get('hostname', settings.rhsso.hostname)
```

Comment:
docblock here please to explain why you're injecting settings.rhsso and what you're expecting to read from it.